hpat/hiframes/pd_dataframe_ext.py

```diff
@@ -1259,6 +1259,39 @@ def lower_mean_dummy(context, builder, sig, args):
     return out_obj._getvalue()
 
 
+@overload_method(DataFrameType, 'median')
+def median_overload(df, axis=None, skipna=None, level=None, numeric_only=None):
+    # TODO: kwargs
+    # TODO: avoid dummy and generate func here when inlining is possible
+    def _impl(df, axis=None, skipna=None, level=None, numeric_only=None):
+        return hpat.hiframes.pd_dataframe_ext.median_dummy(df)
+
+    return _impl
+
+
+def median_dummy(df, n):
+    return df
+
+
+@infer_global(median_dummy)
```

Comment:
Are you sure it is needed? Typing should be done in first part of `overload_method` (lines 1264 in he PR)

Reply:
Isn't this for the dummy while the above is for the actual median method?